pmm-app/src/pmm-dbaas/components/DBCluster/DBCluster.hooks.ts

```diff
@@ -68,13 +65,8 @@ const getClusters = async (kubernetes: Kubernetes[], databaseType: Databases): P
 
     const clusters: DBClusterPayload[] = (r as FulfilledPromiseResult).value?.clusters ?? [];
 
-    const resultClusters = clusters.map(
-      (cluster) => dbClusterService.toModel(
-        cluster,
-        kubernetes[index].kubernetesClusterName,
-        databaseType,
-      ),
-    );
+    // eslint-disable-next-line max-len
+    const resultClusters = clusters.map((cluster) => dbClusterService.toModel(cluster, kubernetes[index].kubernetesClusterName, databaseType));
```

Comment:
Why this change? Was it the IDE? I think it's better like it was to avoid this eslint comments. What do you think?

Reply:
fixed, it was autoformatting from eslint